actions/views/channel.js

```diff
@@ -12,3 +14,15 @@ export function checkAndSetMobileView() {
         });
     };
 }
+
+export function addUsersToChannel(channelId, userIds) {
```

Comment:
This really isn't a view action is it? Not sure it belongs in the views module/folder

Reply:
@jwilander why wouldn't this be a view action? actually I do think everything handle by the views should be a view action.

I mean I can make the change no problem, but I just want to understand the reasoning.

Reply:
A view action is something local only that will only affect the view state of the client. This is adding users to a channel which is affecting the backend state and really doesn't have anything directly to do with view state.

Reply:
alright I'll move it